ui/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.5.1",
+    "@react-leaflet/core": "^1.0.2",
```

Comment:
This command helps to create the production build. However, the spatio-temporal metadata does not work. I have already created the issue [213](https://github.com/o2r-project/o2r-UI/issues/213) to tackle this problem. The only solution working for me at the moment is to downgrade react-leaflet-draw https://github.com/NJaku01/o2r-UI/commit/a8f3159c1fd2fe24c4e4f118cbfcf70a17d2ddca